examples/arithmetics/src/language-server/generated/ast.ts

```diff
@@ -55,7 +55,7 @@ export function isDeclaredParameter(item: unknown): item is DeclaredParameter {
 }
 
 export interface Definition extends AstNode {
-    readonly $container: Module;
+    readonly $container: Definition | Module;
```

Comment:
This seems wrong: a Definition can only be contained in a Module.

Reply:
This is because `Definition` is a container type of `DeclaredParameter` and `Definition`and `DeclaredParameter` are in the same connected component.